### PR TITLE
Add links to rollup PDFs

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Completes [PLAT-958](https://codedotorg.atlassian.net/browse/PLAT-958). Adds a "Printing Options" dropdown link to the script overview page for teachers and turns the "Print Lesson Plan" button on teacher elesson plans into a "Printing Options" dropdown.

Script Overview:
Screen Shot 2021-04-15 at 7 38 03 AM

Lesson Plan:
Screen Shot 2021-04-15 at 7 37 31 AM

## Testing story

added some unit tests, manual testing

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by dmcavoy — APPROVED** [review]
LGTM!

File: dashboard/lib/services/curriculum_pdfs/resources.rb [hunk 1]
```diff
@@ -10,6 +10,20 @@ module CurriculumPdfs
     module Resources
       extend ActiveSupport::Concern
       class_methods do
+        def get_script_resources_pathname(script)
```
Comment (Hamms) [inline dashboard/lib/services/curriculum_pdfs/resources.rb:13]: haha, I just wrote practically this same code for the rake task improvements!

File: dashboard/app/models/script.rb [hunk 2]
```diff
@@ -1913,4 +1915,16 @@ def self.seed_from_json_file(script_name)
   def self.script_json_filepath(script_name)
     "#{script_json_directory}/#{script_name}.script_json"
   end
+
+  def get_pdf_for_script_overview
```
Comment (Hamms) [inline dashboard/app/models/script.rb]: super tiny nit: I think it's worth specifying in the name that these methods get you the _url_ for the pdf, not the pdf itself.

**Review by Hamms — APPROVED** [review]
looks great!

**Pull request merged**